src/check.js

```diff
@@ -1,37 +1,36 @@
-'use strict'
+'use strict';
 
-require('dotenv').config()
 const puppeteer = require('puppeteer');
+const Login = require('./login');
 
-const login = require('./login');
+module.exports = async ({user, password}) => {
 
-(async () => {
     const headless = process.env.INTERFACE !== "true"
-
+    
```

Comment:
extra identations

Reply:
I should have run prettify or add it to prepush on the package.json